tests/phpunit/api/v4/Mock/MockV4ReflectionChild.php

```diff
@@ -7,7 +7,7 @@
  */
 class MockV4ReflectionChild extends MockV4ReflectionBase {
   /**
-   * @inheritDoc
+   * @var array
```

Comment:
@totten this caused a test failure because the test was specifically checking to see if it could resolve `@inheritDoc` annotations.

Reply:
😮 ok. Fixed.